logs/logs.go

```diff
@@ -0,0 +1,65 @@
+package logs
+
+import (
+	jwt "github.com/dgrijalva/jwt-go"
+	dockerClient "github.com/fsouza/go-dockerclient"
+	"github.com/rancherio/host-api/app/common/connect"
+	"github.com/rancherio/host-api/config"
+	"net/http"
+	"strconv"
+)
+
+func GetLogs(rw http.ResponseWriter, req *http.Request) error {
+	urlToken := req.URL.Query().Get("token")
+	token, err := jwt.Parse(urlToken, func(token *jwt.Token) (interface{}, error) {
+		return token, nil
```

Comment:
After reading through the code for what the parse function should return a key. But since mine does not it works because i am not returning an err and the token will always get returned it is just not verified. So Unless I need to reverify the token here then i think it is fine, but I am not sure and is up to you.